src/views/docs-view/server.ts

```diff
@@ -418,9 +418,9 @@ export function getStaticGenerationFunctions<
 			 * Note: If we need more granularity here, we could change this to be
 			 * part of `rootDocsPath` configuration in `src/data/<product>.json`.
 			 */
-			const isPublicContentRepo = !['hcp', 'sentinel', 'terraform'].includes(
-				product.slug
-			)
+			const isHcp = product.slug == 'hcp'
+			const isSentinel = product.slug == 'sentinel'
+			const isPublicContentRepo = !isHcp && !isSentinel
```

Comment:
Since `isHcp` and `isSentinel` aren't used elsewhere, I think we can simplify it to this:
```suggestion
			const isPublicContentRepo = product.slug === 'hcp' && product.slug === 'sentinel'
```
WDYT?

Reply:
I like this! The only thing that gives me pause for thought about this is that this PR is just to revert the code to it's previous state. What if we kept this as-is in this PR and then I borrowed this snippet for my PR to correct the behavior? Would you be good with that?

Reply:
[Thusly](https://github.com/hashicorp/dev-portal/commit/ee172e16c9d8c8cda876731990d1b4f22adf70ec)